ServerData.java

```diff
@@ -0,0 +1,1114 @@
+     * The point40
+     */
+    public void setPoint40(List<Double> point40) {
+        this.point40 = point40;
+    }
+
+    /**
+     *
+     * @return
+     * The point41
+     */
+    public List<Double> getPoint41() {
+        return point41;
+    }
+
+    /**
+     *
+     * @param point41
+     * The point41
+     */
+    public void setPoint41(List<Double> point41) {
+        this.point41 = point41;
+    }
+
+    /**
+     *
+     * @return
+     * The point42
+     */
+    public List<Double> getPoint42() {
+        return point42;
+    }
+
+    /**
+     *
+     * @param point42
+     * The point42
+     */
+    public void setPoint42(List<Double> point42) {
+        this.point42 = point42;
+    }
+
+    /**
+     *
+     * @return
+     * The point43
+     */
+    public List<Double> getPoint43() {
+        return point43;
+    }
+
+    /**
+     *
+     * @param point43
+     * The point43
+     */
+    public void setPoint43(List<Double> point43) {
+        this.point43 = point43;
+    }
+
+    /**
+     *
+     * @return
+     * The point44
+     */
+    public List<Double> getPoint44() {
+        return point44;
+    }
+
+    /**
+     *
+     * @param point44
+     * The point44
+     */
+    public void setPoint44(List<Double> point44) {
+        this.point44 = point44;
+    }
+
+    /**
+     *
+     * @return
+     * The point45
+     */
+    public List<Double> getPoint45() {
+        return point45;
+    }
+
+    /**
+     *
+     * @param point45
+     * The point45
+     */
+    public void setPoint45(List<Double> point45) {
+        this.point45 = point45;
+    }
+
+    /**
+     *
+     * @return
+     * The point46
+     */
+    public List<Double> getPoint46() {
+        return point46;
+    }
+
+    /**
+     *
+     * @param point46
+     * The point46
+     */
+    public void setPoint46(List<Double> point46) {
+        this.point46 = point46;
+    }
+
+    /**
+     *
+     * @return
+     * The point47
+     */
+    public List<Double> getPoint47() {
+        return point47;
+    }
+
+    /**
+     *
+     * @param point47
+     * The point47
+     */
+    public void setPoint47(List<Double> point47) {
+        this.point47 = point47;
+    }
+
+    /**
+     *
+     * @return
+     * The point48
+     */
+    public List<Double> getPoint48() {
+        return point48;
+    }
+
+    /**
+     *
+     * @param point48
+     * The point48
+     */
+    public void setPoint48(List<Double> point48) {
+        this.point48 = point48;
+    }
+
+    /**
+     *
+     * @return
+     * The point49
+     */
+    public List<Double> getPoint49() {
+        return point49;
+    }
+
+    /**
+     *
+     * @param point49
+     * The point49
+     */
+    public void setPoint49(List<Double> point49) {
+        this.point49 = point49;
+    }
+
+    /**
+     *
+     * @return
+     * The point50
+     */
+    public List<Double> getPoint50() {
+        return point50;
+    }
+
+    /**
+     *
+     * @param point50

```

Comment:
I checked if the markers were being placed on the map,if a blue dot was
shown at the current location and if the initial and final markers were
being updated. However I couldn't think of a way to check if the rest part
was working properly. Also what about the destination url for both these
requests? For now I just put one url.
I will push this code in a while. Tomorrow I will work on camera in the map
and I was thinking that since this is mostly done I will pull out a map a
iitm and try to identify nodes so we can decide on the number of devices we
will need.
On Saturday, May 21, 2016, Kunal Grover <notifications@github.com> wrote:
> In ServerData.java:
>
>> + public void setPoint49(List<Double> point49) {
>> + this.point49 = point49;
>> + }
>> +
>> + /**
>> + *
>> + * @return
>> + * The point50
>> + */
>> + public List<Double> getPoint50() {
>> + return point50;
>> + }
>> +
>> + /**
>> + *
>> + * @param point50
>
> Sounds good. Have you tested your app?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub<
https://ci6.googleusercontent.com/proxy/8w_4anAKOgcXa9cF0bvLLDi1ueC8syBYjJag8alFEFGMs_SNMqB0vzzDN5rsAHT2C4VwLcd_tyHjEIRf0CuHTVZFGxGWD-yByjWgDemW8UGhg3uFsDduGnfJBl6YSkCjUZbS_JmbCBE7uuA_-k5EQ3oCj2q6Jg=s0-d-e1-ft#https://github.com/notifications/beacon/ARhz6DeQH7did0s2p8pfds9hxgI1Tkhoks5qDxjkgaJpZM4IjuFm.gif
>

-- 
Karteek Dhara
First year student,
Dual Degree | Dept. of Electrical Engineering
Indian Institute of Technology, Madras